Wishlist/wishlist.html

```diff
@@ -0,0 +1,260 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- Stylesheets -->
+    <link rel="stylesheet" href="https://halcyon-ui.netlify.app/halcyon.css">
+    <link rel="stylesheet" href="../css/main.css">
+    <link rel="stylesheet" href="../css/wishlist.css">
+    <title>Browse Wishlist | Halcyon Skin Store</title>
+</head>
+
+<body>
+    <!-- Navigation Top -->
+    <section id="header">
+        <nav class="store-nav-bar">
+            <div class="store-logo-box">
+                <a href="./"><img src="/assets/HalcyonStoreLogo.png" alt="logo" class="store-logo"></a>
+            </div>
+            <div class="store-nav">
+                <ul class="store-nav-links ul-no-decor display-flex">
+                    <li class="search-bar">
+                        <input type="search" placeholder="Search our skincare store ">
+                        <label class="search-bar-icon">
+                            <span class="fas fa-search"></span>
+                        </label>
```

Comment:
here you can give input an id and label for tag to connect them  for accessibility purpose. like if focus comes on search input it can be read using that connection for screen readers.

Reply:
Will fix this at the end in navbar thanks!